src/Uno.UI/Extensions/MathEx.cs

```diff
@@ -29,5 +51,15 @@ public static double ToRadians(double angleDegree)
 		/// </summary>
 		public static double ToDegree(double angleRadian)
 			=> angleRadian / (Math.PI / 180.0);
+
+		/// <summary>
+		/// Tests if two values are equal to within the specified error.
+		/// </summary>
+		/// <param name="value1">First value</param>
+		/// <param name="value2">Second value</param>
+		/// <param name="delta">Permitted error</param>
+		/// <returns>True if the difference is less than the permitted error, false otherwise</returns>
+		public static bool ApproxEqual(double value1, double value2, double delta = 1e-9)
```

Comment:
Should it be `double.Epsilon` ?

Reply:
No, using `double.Epsilon` as a delta is generally a bad idea. Floating-point imprecision can be larger than `Epsilon`.